dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -0,0 +1,1097 @@
+                "  FROM airline, airlineref \n" +
+                "  WHERE airline.UniqueCarrier = airlineref.CODE AND  Origin like '%SFO%' AND WeatherDelay > 0 \n" +
+                "  GROUP BY DESCRIPTION \n" +
+                "  limit 20 ";
+        Row[] result = snc.sql(query).collect();
+        Log.getLogWriter().info("SS - Qyery executed successfully and query result is ::" + ((GenericRow) result[0]).get(0));
+    }
+
+    /**
+     * This method updates the description for a code in row table. Suppose a particular Airline company say 'Delta Air Lines Inc.' re-brands itself as 'Delta America'
+     * --- the airline code can be updated in the row table
+     */
+
+    public static void HydraTask_updateRowTable() {
+        String query1 = "UPDATE AIRLINEREF SET DESCRIPTION='Delta America' WHERE CODE='DL' ";
+        Row[] result = snc.sql(query1).collect();
+        Log.getLogWriter().info("SS - Qyery executed successfully");
+        String query2 = "SELECT * FROM AIRLINEREF WHERE CAST(CODE AS VARCHAR(25))='DL'";
+        Row[] result1 = snc.sql(query2).collect();
+//        Long expectedValue = Long.parseLong("DL");
+//        Assert.assertEquals(expectedValue, ((GenericRow) result[0]).get(0));
+        Log.getLogWriter().info("SS - Qyery executed successfully and query result is ::" + ((GenericRow) result1[0]).get(0));
+    }
+
+    protected String getDataLocation() {
+        String scriptPath = null;
+        try {
+            hd = getHostDescription();
+            scriptPath = hd.getGemFireHome() + sep + ".." + sep + "snappy"
+                    + sep + "quickstart" + sep + "data";
+            Log.getLogWriter().info("SS - scriptPath is ::" + scriptPath);
+        } catch (Exception e) {
+            throw e;
+        }
+        return scriptPath;
+    }
+
+    protected String getDataFrameDataLocation() {
+        String scriptPath = null;
+        try {
+            hd = getHostDescription();
+            scriptPath = hd.getGemFireHome() + sep + ".." + sep + ".."
+                    + sep + ".." + sep + "tests-common" + sep + "src" + sep + "main" + sep + "resources";
+            Log.getLogWriter().info("SS - scriptPath is ::" + scriptPath);
+        } catch (Exception e) {
+            throw e;
+        }
+        return scriptPath;
+    }
+
+    protected HostDescription getHostDescription() {
+        hd = TestConfig.getInstance()
+                .getClientDescription(RemoteTestModule.getMyClientName())
+                .getVmDescription().getHostDescription();
+        sep = hd.getFileSep();
+        Log.getLogWriter().info("SS - Inside getHostDescription - gemfireHome is ::" + hd.getGemFireHome());
+        return hd;
+    }
+
+    protected String getScriptLocation(String scriptName) {
+        String scriptPath = null;
+        try {
+            hd = getHostDescription();
+            scriptPath = hd.getGemFireHome() + sep + ".." + sep + "snappy"
```

Comment:
Added this feature at required places for now. e.g. for getting data location while creating the table. Will do the same changes for scripts locations later on.